spec/services/pull_request/parse_repository_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe 'PullRequest::ParseRepository' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.